### DISCUSSION TITLE
Ideas for an alternative Faster than AsyncStorage

### DISCUSSION THREAD

**Discussion by otekdo** [infinitered/ignite#2270] [Ideas]
Hi guys, Does anyone have any experience using ``react-native-mmkv``?

Or are there plans for using this tool as the default key/value storage for this awesome boilerplate! 🙂

This is an alternative which claims to be ``30x`` faster than ``AsyncStorage``!
https://github.com/mrousavy/react-native-mmkv
![image](https://user-images.githubusercontent.com/65221868/196025987-1535c696-5ed3-41a4-902d-788254a77133.png)

**Comment by frankcalise** [comment]
@otekdo Thanks for the discussion! We actually have plans to create a recipe for you to convert over from AsyncStorage to react-native-mmkv within the [Ignite Cookbook](https://infinitered.github.io/ignite-cookbook/docs/intro).

Although we haven't written the tutorial yet, someone has dropped their own notes in the issue we're tracking [over here](https://github.com/infinitered/ignite-cookbook/issues/9) which may help you.

Hope this helps

**Reply by otekdo** [reply]
Super awesome!
